mulesoft.properties

```diff
@@ -0,0 +1,17 @@
+category=Coverage
+description=Coverage analyzer for Mulesoft Workflows
+homepageUrl=https://github.com/UltimateSoftware/sonar-mulesoft
+devVersion=
+name=Mulesoft
+publicVersions=0.5.1
+archivedVersions=
+privateVersions=
+
+defaults.mavenGroupId=com.ultimatesoftware.sonar.plugins.mulesoft
+defaults.mavenArtifactId=mulesoftplugin
+
+0.5.1.description=Initial release, includes basic support for analyzing Mulesoft test coverage reports.
+0.5.1.sqlVersions=[7.3,*]
```

Comment:
`*` is not valid as the end of the compatibility range. Use `LATEST` instead

Reply:
This is fixed.